### PR TITLE
Customizing distance sensor to add orientation for multiple sensor support

### PR DESCRIPTION
**Pull request by prokas-nikos**
Customized telemetry.h and telementry_impl.cpp to add the orientation parameter to telemetry.distance_sensor to add multisensor support readings on distance_sensor

Depends on https://github.com/mavlink/MAVSDK-Proto/pull/321 :heavy_check_mark:

### CODE REVIEW

**Comment by JonasVautherin** [comment]
I think you need to fix the style in your PR (running `./tools/fix_style.sh .`) and remove the unwanted files (I see a bit "output.log" file that shouldn't be here)

File: examples/distance_sensor/distance_sensor.cpp [hunk 1]
```diff
@@ -0,0 +1,69 @@
+//
+// Simple example to demonstrate how to imitate a smart battery.
```
Comment (JonasVautherin) [inline examples/distance_sensor/distance_sensor.cpp:2]: Smart battery? :grin:

Comment (prokas-nikos) [reply examples/distance_sensor/distance_sensor.cpp]: Right 😁. Updated also the comments 😁

File: src/mavsdk/plugins/telemetry/telemetry_impl.cpp [hunk 2]
```diff
@@ -1451,6 +1451,213 @@ void TelemetryImpl::process_distance_sensor(const mavlink_message_t& message)
     distance_sensor_struct.current_distance_m =
         static_cast<float>(distance_sensor_msg.current_distance) * 1e-2f; // cm to m
 
+    MavSensorOrientation orientation =
```
Comment (JonasVautherin) [inline src/mavsdk/plugins/telemetry/telemetry_impl.cpp:1454]: Can we maybe put that whole big switch block in a separate function? Like `extractOrientation(distance_sensor_msg);`?

**Review by JonasVautherin — COMMENTED** [review]
Yes that's looking better!

**Comment by julianoes** [comment]
Looks like some namespacing issue in the build?

**Comment by prokas-nikos** [comment]
Yes sorry for that. I fixed it and its building now. I checked it locally.

**Comment by julianoes** [comment]
@prokas-nikos sorry for the delay. I'm getting back to the merge queue here soon.

**Comment by julianoes** [comment]
Sorry, I dropped the ball on this. Getting it in now.

**Review by julianoes — DISMISSED** [review]

**Pull request merged**